apps/web/public/static/locales/en/common.json

```diff
@@ -771,7 +776,6 @@
   "api_keys": "API Keys",
   "api_key_modal_subtitle": "API keys allow you to make API calls for your own account.",
   "api_keys_subtitle": "Generate API keys to use for accessing your own account.",
-  "generate_new_api_key": "Generate new API key",
```

Comment:
To be consistent with Webhooks section in Developer tab in Settings, I removed the text along side the "plus" icon